pkg/controllers/metrics/kluctldeployment_controller.go

```diff
@@ -9,14 +9,15 @@ import (
 const (
 	KluctlDeploymentControllerSubsystem = "kluctldeployments"
 
-	DeploymentIntervalKey = "deployment_interval_seconds"
-	DryRunEnabledKey      = "dry_run_enabled"
-	LastObjectStatusKey   = "last_object_status"
-	PruneEnabledKey       = "prune_enabled"
-	DeleteEnabledKey      = "delete_enabled"
-	SourceSpecKey         = "source_spec"
-	GitSourceSpecKey      = "git_source_spec"
-	OciSourceSpecKey      = "oci_source_spec"
+	DeploymentIntervalKey  = "deployment_interval_seconds"
+	DryRunEnabledKey       = "dry_run_enabled"
+	LastObjectStatusKey    = "last_object_status"
+	LastDeployStartTimeKey = "last_deploy_start_time"
```

Comment:
There seems to be naming conventions for prometheus that suggest to use `_timestamp_seconds` as a suffix. See  https://prometheus.io/docs/practices/naming/